habitat-lab/requirements.txt

```diff
@@ -14,3 +14,5 @@ imageio>=2.2.0
 imageio-ffmpeg>=0.2.0
 scipy>=1.0.0
 tqdm>=4.0.0
+# Home robot dependency for velocity controllers
+home-robot @ git+ssh://git@github.com/facebookresearch/home-robot.git@habitat-challenge-2023#subdirectory=src/home_robot
```

Comment:
Not a fan of having this dependency here. Can we make it an optional dependency ?

Reply:
@exhaustin

Reply:
#1183

Reply:
Please remove from this file since the additional dependency is added in setup.py as per #1183